src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 101;
+    private Ball ball = new Ball();
+
+    public Ball getRandomBall() {
+        ball.setColor();
+        ball.setNumber(new Random().nextInt(BOUND));
```

Comment:
Creating a new instance of 'Random' every time the method is called is inefficient. A single instance should be created at the class level and reused.